src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,11 +1,22 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String baseOfBulldozer;
+
+    public Bulldozer(String name, String baseOfBulldozer, String color) {
+        this.baseOfBulldozer = baseOfBulldozer;
```

Comment:
```suggestion
    public Bulldozer(String name, String baseOfBulldozer, String color) {
        super(name, color);
        this.baseOfBulldozer = baseOfBulldozer;
```

немає сенсу передавати в конструктор параметри, які ти не присвоюєш полям екземпляру класу

Reply:
А зрозумів краще використати super()